package.json

```diff
@@ -23,9 +23,10 @@
   "main": "lib/prepack-node.js",
   "browser": "lib/prepack-standalone.js",
   "scripts": {
-    "build": "babel src --out-dir lib --source-maps && yarn build-bundle",
+    "build": "yarn build-lib && yarn build-bundle",
     "build-scripts": "babel scripts --out-dir lib --source-maps",
     "build-bundle": "webpack-cli",
+    "build-lib": "babel src --out-dir lib --source-maps",
```

Comment:
Maybe call this build-prepack?

Reply:
I agree, `build-prepack` makes more sense given how many build commands we have already.

Reply:
Thanks, I couldn't come up with a better name!